sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/AppDestination.kt

```diff
@@ -0,0 +1,29 @@
+package co.nimblehq.sample.compose.ui
+
+import androidx.navigation.*
+
+const val KEY_ID = "id"
+
+sealed class AppDestination(val route: String = "") {
+
+    open val arguments: List<NamedNavArgument> = emptyList()
+
+    open var destination: String = route
+
+    //====================================================//
+
+    object Up : AppDestination()
+
+    object Home : AppDestination("home")
```

Comment:
Would it be good to create `const` for these?

Reply:
@Tuubz with `sealed class`, it's unnecessary, like what we do with enums 😛 Detekt will not report the issues ;)
```
enum class AppDestination(value: String) {
    HOME("home"),
    DETAIL("detail")
}
```